Sum.py

```diff
@@ -0,0 +1,5 @@
+sum = 0
+for i in range(1, 101):
+    if i % 3 != 0 and i % 7 != 0 and i % 13 != 0:
+        sum = sum+i
+print(sum)
```

Comment:
When you are printing anything on console, you should print with a message along with result.